cmd/utils/flags.go

```diff
@@ -1214,7 +1215,7 @@ func setStaticPeers(ctx *cli.Context, cfg *p2p.Config) {
 		urls = libcommon.CliString2Array(ctx.String(StaticPeersFlag.Name))
 	} else {
 		chain := ctx.String(ChainFlag.Name)
-		urls = params.StaticPeerURLsOfChain(chain)
+		urls = params2.StaticPeerURLsOfChain(chain)
```

Comment:
I don't know if I like have `params` and `params2` being in radically different locations. This seems to me to create even more package fragmentation. If the reason for this move is to avoid import cycles, I would then suggest to move the entire `github.com/erigontech/erigon/params` to `erigon-lib`.

Reply:
Fair enough. I would just also like to add that to it would be good to minimize the time that the repository structure remains in this intermediate state. 

 In fact, I think in this case it may have even been easier to move the entire `ithub.com/erigontech/erigon/params` in one go, since it would avoid having to deal with `params2` altogether.